polyfill/lib/calendar.mjs

```diff
@@ -414,11 +415,11 @@ class OneObjectCache {
   constructor(cacheToClone = undefined) {
     this.map = new Map();
     this.calls = 0;
-    this.now = globalThis.performance ? globalThis.performance.now() : Date.now();
     this.hits = 0;
     this.misses = 0;
+    this.now = globalThis.performance ? globalThis.performance.now() : Date.now();
     if (cacheToClone !== undefined) {
-      let i = cacheToClone.length;
+      let i = 0;
```

Comment:
This was a bug that @12wrigja  found in temporal-polyfill that resulted in the cache sometimes holding fewer items than normal; if the cache is cloned from another one then each existing entry was counted twice..

Reply:
handled in #1983